README.md

```diff
@@ -124,52 +131,59 @@ Find more examples in the [examples dir](examples).
 The default jnlp agent image used can be customized by adding it to the template
 
 ```groovy
-containerTemplate(name: 'jnlp', image: 'jenkins/inbound-agent:4.3-4-alpine', args: '${computer.jnlpmac} ${computer.name}'),
+containerTemplate(name: 'jnlp', image: 'jenkins/inbound-agent:4.7-1', args: '${computer.jnlpmac} ${computer.name}'),
 ```
 
 or with the yaml syntax
 
-```
+```yaml
 apiVersion: v1
 kind: Pod
 spec:
   containers:
   - name: jnlp
-    image: 'jenkins/inbound-agent:4.3-4-alpine'
+    image: 'jenkins/inbound-agent:4.7-1'
     args: ['\$(JENKINS_SECRET)', '\$(JENKINS_NAME)']
 ```
 
-### Container Group Support
+### Multiple containers support
 
-Multiple containers can be defined for the agent pod, with shared resources, like mounts. Ports in each container can be accessed as in any Kubernetes pod, by using `localhost`.
+Multiple containers can be defined for the agent pod, with shared resources, like mounts. Ports in each container can
+be accessed as in any Kubernetes pod, by using `localhost`.
 
-The `container` statement allows to execute commands directly into each container.
+The `container` step allows executing commands into each container.
+
+---
+**Note**
+Due to implementation constraints, there can be issues when executing commands in different containers if they run using different uids.
+It is recommended to use the same uid across the different containers part of the same pod to avoid any issue.
+---
 
 ```groovy
 podTemplate(containers: [
-    containerTemplate(name: 'maven', image: 'maven:3.3.9-jdk-8-alpine', ttyEnabled: true, command: 'cat'),
-    containerTemplate(name: 'golang', image: 'golang:1.8.0', ttyEnabled: true, command: 'cat')
+    containerTemplate(name: 'maven', image: 'maven:3.8.1-jdk-8', command: 'sleep', args: '99d'),
+    containerTemplate(name: 'golang', image: 'golang:1.16.5', command: 'sleep', args: '99d')
```

Comment:
Be sure to test uses of `99d` in particular images since some non-GNU versions of `sleep` do not support it. That is why I have been using `9999999` as a more portable alternative.

Reply:
Looks good on busybox-based distros (like Alpine) so I would say we're good with that.